README.md

```diff
@@ -97,11 +97,11 @@ $ eksctl create cluster
 $
 ```
 
```

Comment:
The following changes were necessary as the original `mkdocs` PRs merged the different introduction pages into one. It's a change I agree with - the pages were all pretty short.